docs/quick-start-guide.md

```diff
@@ -54,7 +54,8 @@ CarbonData can be integrated with Spark,Presto and Hive Execution Engines. The b
 ### Hive
 [Installing and Configuring CarbonData on Hive](https://github.com/apache/carbondata/blob/master/docs/hive-guide.md)
 
-
+### Alluxio
```

Comment:
As mentioned above, we may need to adjust the location for this section.

Reply:
This is not the location for alluxio. need to add a separate section to describe storages supported

Reply:
ok, added, please check it